model/ftn/w3iogomd.ftn

```diff
@@ -2535,7 +2541,8 @@
           CALL EXTCDE ( 1 )
         END IF
 !
-      IF ( IPASS.EQ.1 ) THEN
+!INLINE      IF ( IPASS.EQ.1 ) THEN
```

Comment:
What is the INLINE switch? 
Need documentation updates for added new switch, in addition updates need to be made to the makefile for adding a new switch for the clean and make procedures. See https://github.com/NOAA-EMC/WW3/wiki/FAQs-page#how-to-add-a-new-switch-to-makefile

Reply:
@jessica good eye, but this is a comment that needs to be removed (not a new !/INLINE switch). Will be removed.

Reply:
Spurious line removed.

Reply:
@RobertoPadilla-NOAA added to thread for awareness